composer.json

```
@@ -6,13 +6,19 @@
     "require": {
         "php": "^8",
```

I think the mininum supported version is 8.1

```suggestion
        "php": "^8.1",
```

---

Add `silverstripe/framework: ^5.2` too